### DISCUSSION TITLE
Scroll on Jump

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#526] [Show and tell]
I didn't make this but I found the package [scroll-on-jump](https://codeberg.org/ideasman42/emacs-scroll-on-jump) and using the below setup I got it working on meow and it looks great!

```
;; Scroll on jump, https://codeberg.org/ideasman42/emacs-scroll-on-jump
(scroll-on-jump-advice-add meow-undo)
(scroll-on-jump-advice-add meow-search)
(scroll-on-jump-advice-add meow-visit)
(scroll-on-jump-advice-add meow-line)
(scroll-on-jump-advice-add meow-beginning-of-thing)
(scroll-on-jump-advice-add meow-end-of-thing)
(scroll-on-jump-advice-add meow-pop-selection)

(setq scroll-on-jump-curve 'smooth-in)
```

It basically makes it so there is a scroll for each of those commands rather than an immediate jump. I personally think it looks really nice!

**Comment by ideasman42** [comment]
( `scroll-on-jump` author here), I've been checking out meow.
Now scroll-on-jump hides the selection while jumping so it doesn't flicker while scrolling with `meow-visit` & `meow-search` .

**Reply by chookity-pokk** [reply]
That's great news! I love your packages and glad you found meow. It's maybe my favorite Emacs package.
